ios/REAUtils.h

```diff
@@ -0,0 +1,11 @@
+#import <Foundation/Foundation.h>
+#import <React/RCTConvert.h>
+#import <React/RCTLog.h>
+
+#ifndef REA_LOG_ERROR_IF_NIL
+
+#define REA_LOG_ERROR_IF_NIL(value, errorMsg, nodeID) ({\
```

Comment:
Can we put in the same place as the rest of macros?

Reply:
Where is it? Do you mean REATransition.h? There is one macro defined.

Reply:
ah nvm.